lewis/adapters/modbus.py

```diff
@@ -546,8 +549,9 @@ def handle_close(self):
 
 @has_log
 class ModbusServer(asyncore.dispatcher):
-    def __init__(self, host, port, interface=None):
+    def __init__(self, host, port, interface, lock):
```

Comment:
Tiny nitpick, it would be a little better if the lock was names consistently throughout (`lock` or `device_lock` or `io_lock`, etc... just consistent everywhere).